kie-dmn-core/src/main/java/org/kie/dmn/core/util/Msg.java

```diff
@@ -0,0 +1,12 @@
+package org.kie.dmn.core.util;
+
+
+public class Msg extends org.kie.dmn.feel.util.Msg {
```

Comment:
I think we don't need to extend it. Just create a standalone class for dmn-core.